### DISCUSSION TITLE
When using multiple #Rules, the pipe characters break validation.

### DISCUSSION THREAD

**Discussion by citricguy** [livewire/livewire#6326] [7. Bugs (archived)]
### Livewire version

Beta .8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Consider the following:

```
    #[Rule('nullable|url:http,https')]
    #[Rule('min:3')]
    public $title;
```

When validating, the following error is returned:

` Method Illuminate\Validation\Validator::validateNullable|url does not exist.`

However, the following work just fine.

```
//This example has everything in a single rule.
    #[Rule('nullable|url:http,https|min:3')]
    public $title;

// AND

//This example has each validation in its own rule. (no pipe seperation)
    #[Rule('nullable')]
    #[Rule('url:http,https')]
    #[Rule('min:3')]
    public $title;
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment]
Your example with a pipe character for multiple `#Rule` attributes is equivalent to the following Laravel example:
```php
$request->validate([
    'title' => [
        'nullable|url:http,https',
        'min:3',
    ],
]);
```
Even in Laravel, that isn't a supported usage, but I agree with you that it's confusing.

**Reply by citricguy** [reply]
Ahh, I didn't realize that it was an all or none feature. I would assume that if this is even considered an issue, it would need to be addressed upstream?
